src/utils/checkForBraveBrowser.ts

```diff
@@ -0,0 +1,16 @@
+// https://stackoverflow.com/questions/36523448/how-do-i-tell-if-a-user-is-using-brave-as-their-browser
+let isBraveBrowser = false
+;(() => {
```

Comment:
this is shorter.
```
;(async () => {
  isBraveBrowser = (navigator.brave && (await navigator.brave.isBrave())) || false
})()
```

Reply:
🙌

Reply:
I mean you can write that code directly in IIFE, no need to create function `check` . so there is only 1 line in IIFE function

Reply:
Yeah, agree